supportedBackends/htcondor/src/main/scala/cromwell/backend/impl/htcondor/HtCondorJobExecutionActor.scala

```diff
@@ -99,7 +135,7 @@ class HtCondorJobExecutionActor(override val jobDescriptor: BackendJobDescriptor
         val matcher = pattern.matcher(line)
```

Comment:
There is no harm in existing code but for consistency if possible you could try: ` val Job = HtCondorCommands.SubmitOutputPattern.r ; line match { case Job(jobId, clusterId) => ...   case _ => exception()... } `